dask/base.py

```diff
@@ -571,15 +571,15 @@ def compute(*args, **kwargs):
 
 def visualize(*args, **kwargs):
     """
-    Visualize several dask graphs at once.
+    Visualize a dask low level graph.
```

Comment:
**THIS** function does indeed take as many graphs as you like, the original docstring was correct. It's only for the `self.visualize` that the graph in question is already implied and there is no way to show more.

Reply:
Oh! Didn't know that :heavy_check_mark: